entity-framework/core/modeling/value-comparers.md

```diff
@@ -16,43 +16,34 @@ uid: core/modeling/value-comparers
 
 ## Background
 
-EF Core needs to compare property values when:
+In EF, change tracking means that EF automatically determines what changes were performed by the application on a loaded entity instance, so that those changes can be saved back to the database when <xref:Microsoft.EntityFrameworkCore.DbContext.SaveChanges%2A> is called. EF usually performs this by taking a *snapshot* of the instance when it's loaded from the database, and *comparing* that snapshot to the instance handed out to the application.
```

Comment:
Do we say EF or EF Core? I have been consistently saying EF Core.

Reply:
We might not be consistent... In any case, changing the entire page to always say EF.